modules/swagger-codegen/src/main/resources/python/swagger.mustache

```diff
@@ -131,10 +131,14 @@ class ApiClient(object):
       if isinstance(obj, dict):
         objDict = obj
       else:
-        objDict = obj.__dict__
+        # Convert model obj to dict except attributes `swaggerTypes`, `attributeMap`
+        # and attributes which value is not None.
+        # Convert attribute name to json key in model definition for request.
+        objDict = {obj.attributeMap[key]: val
```

Comment:
In request, we should convert the model attribute name to the json key in model definition.
So the key in request json is same with the key in model definition.